python/lib/dcoscli/dcoscli/data/help/marathon.txt

```diff
@@ -17,6 +17,8 @@ Usage:
     dcos marathon app kill [--scale] [--host=<host>] <app-id>
     dcos marathon app update [--force] <app-id> [<properties>...]
     dcos marathon app version list [--max-count=<max-count>] <app-id>
+    dcos marathon delay reset <app-id>|<pod-id>
+    dcos marathon delay show <app-id>|<pod-id>
```

Comment:
I believe the `|` in `<app-id>|<pod-id>` will restrict to 1 argument... and the underlying code will only get 1.   I think we need to be generic here and provide details in the help section of this file.

perhaps a <app-pod-id> with details below

Reply:
@kensipe
What we are passing here is a `runSpecId`. But there is no `runSpecId` (yet) in the help menu. Creating `runSpecId` would address all other concerns you mentioned in trailing comments, but I am not sure if we want to do that. What else do you suggest here?